tn40.c

```diff
@@ -434,16 +434,18 @@ static enum PHY_TYPE bdx_phy_init(struct bdx_priv *priv)
 	    bdx_get_phy_by_id(pdev->vendor, pdev->device,
 			      pdev->subsystem_device);
 
-	if (phy_type == PHY_TYPE_NA)
+	if (phy_type == PHY_TYPE_NA){
+		dev_info(&priv->pdev->dev, "No PHY for subsystem_device [%4x:%4x:%4x]\n", pdev->vendor, pdev->device, pdev->subsystem_device);
 		return PHY_TYPE_NA;	/* NIC definition has no PHY. */
-
+    }
 	bdx_mdio_set_speed(priv->pBdxRegs, MDIO_SPEED_1MHZ);
```

Comment:
Indent with a hard tab to follow the codestyle in the rest of the file. Also one more occurance below. (minor, doesn't hinder merge)